Cargo.toml

```diff
@@ -9,7 +9,7 @@ readme = "README.md"
 homepage = "https://github.com/snapview/tungstenite-rs"
 documentation = "https://docs.rs/tungstenite/0.11.1"
 repository = "https://github.com/snapview/tungstenite-rs"
-version = "0.11.1"
+version = "0.12.0"
```

Comment:
The master has been already bumped to 0.12.0, please rebase and make this a bump from 0.12.0 to 0.13.0. (I believe a bump is necessary as this commit introduces an incompatible change to the library API).